tests/test_flags.py

```diff
@@ -6,28 +6,49 @@
 
 
 class TestTriggers(unittest.TestCase):
-    @mock.patch('charmhelpers.core.unitdata.kv')
-    def test_triggers(self, kv):
+    def setUp(self):
+        self.kv_p = mock.patch('charmhelpers.core.unitdata.kv')
+        kv = self.kv_p.start()
         kv.return_value = MockKV()
 
+    def tearDown(self):
+        self.kv_p.stop()
```

Comment:
Personally, I would have done self.addCleanup(self.kv_p.stop) in the setUp function, and you could even avoid storing the patch object as an attribute entirely.